Lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -1126,6 +1124,13 @@ namespace Js
 #endif
       isByteCodeDebugMode(false)
     {
+        if (nestedCount > 0)
```

Comment:
Is the nestedArray only needed for FunctionBody? 
Is it needed by the ParseableFunctionInfo when it is still in defer parsed state?